core/mrc.js

```diff
@@ -275,21 +275,7 @@ exports.getModule = class mrcModule extends MenuModule {
             const chatLogView = this.viewControllers.mrcChat.getView(
                 MciViewIds.mrcChat.chatLog
             );
-            const messageLength = stripMciColorCodes(msg).length;
-            const chatWidth = chatLogView.dimens.width;
-            let padAmount = 0;
-            let spaces = 2;
-
-            if (messageLength > chatWidth) {
-                padAmount = chatWidth - (messageLength % chatWidth) - spaces;
-            } else {
-                padAmount = chatWidth - messageLength - spaces;
-            }
-
-            if (padAmount < 0) padAmount = 0;
-
-            const padding = ' |00' + ' '.repeat(padAmount);
-            chatLogView.addText(pipeToAnsi(msg + padding));
+            chatLogView.addText(pipeToAnsi(msg));
```

Comment:
I'm not sure this will work correctly, I'll have to do some testing. This MR just got a bit biggerr

Reply:
I have the latest client and it worked fine on my end. All the padding appeared to be from an older way of drawing the screen which appeared to have changed since it was first implemented (assumption here). On my end, it was working pretty well without it and fixed an issue with text wrapping on certain cases. Let me know if there is anything I missed.